marshal/message_test.go

```diff
@@ -40,6 +43,13 @@ func (s *MessageSuite) TestMessageEncode(c *C) {
 		ProposedCurrentOffset: 9,
 	}
 	c.Assert(cm.Encode(), Equals, "ClaimingMessages/4/2/ii/cl/gr/t/3/9")
+
+	rg := msgReleaseGroup{
+		// Base message GroupID identifies which consumer group to release.
+		msgBase:       base,
+		MsgExpireTime: time.Time{},
+	}
+	c.Assert(rg.Encode(), Equals, "ReleaseGroup/4/2/ii/cl/gr/t/3/Mon Jan  1 00:00:00 UTC 0001")
```

Comment:
Yeah.. is your issue with the whitespace? Having to pass a zero struct for tests?